src/configuration.staging.json

```diff
@@ -0,0 +1,54 @@
+{
+  "teamIds": [],
+  "groups": [],
+  "environment": "staging",
```

Comment:
"testing" is much more common than "staging". Should we change the names here too are do you think this is mostly for us?

Reply:
Right. yeah.

Reply:
This is also where I initially wanted to support custom profile names that would be resolved from `appRoot.path`.